### PR TITLE
490: `PID` Documentation in IG

### PR DESCRIPTION
**Pull request by halprin**
# `PID` Documentation in IG

Added the `PID` documentation to the IG.  This documents the fields we support.  I updated the cardinality and the datatypes too.

## Issue

#490

## Checklist

- [n/a] I have added tests to cover my changes
- [n/a] I have added logging where useful (with appropriate log level)
- [n/a] I have added JavaDocs where required
- [x] I have updated the documentation accordingly

**Note**: You may remove items that are not applicable

### CODE REVIEW

**Review by sfradkin — APPROVED** [review]
this all looks reasonable

**Review by sfradkin — CHANGES_REQUESTED** [review]
Actually, on second thought...

Please wait until #505 is merged and then add a Provenance entry describing the changes.

File: ig/input/fsh/nbs-pid-segment.fsh [hunk 1]
```diff
@@ -3,23 +3,29 @@ Id: segment-pid-logical-model
...
 Description: "The PID Segment"
 * segmentType 1..1 SU string "PID"
-* setId 1..1 SU string "PID-1 a sequence number"
-* patientId 0..1 SU string "a patient id (retained for backward compatibility, but should be blank"
-* patientIdList 1..1 SU string "PID-3 a list of patient identifiers"
-* alternatePatientId 0..1 SU string "alternate patient id, should be blank"
-* patientName 1..1 SU string "PID-5 patient name"
-* mothersMaidenName 0..1 SU string "PID-6 mothers maiden name"
-* birthDateTime 1..1 SU string "PID-7 birth date and time"
+* setId 1..1 SU positiveInt "PID-1 a sequence number"
```
Comment (basiliskus) [inline ig/input/fsh/nbs-pid-segment.fsh:6]: Is the first letter in the sentence meant to be capitalized (i.e. `A sequence number`)? I'm not familiar with the format, so I'm not sure

Comment (tjohnson7021) [reply ig/input/fsh/nbs-pid-segment.fsh:6]: I think that's a formatting choice for us, but we should probably be uniform in our choice - I'll clean this up on the branch and make a commit.

**Review by sfradkin — APPROVED** [review]
Thanks for the changes.  This looks good.

**Pull request merged**